llvm/include/llvm/CodeGen/AsmPrinter.h

```diff
@@ -208,6 +209,9 @@ class AsmPrinter : public MachineFunctionPass {
   std::vector<HandlerInfo> Handlers;
```

Comment:
Personal note: `HandlerInfo` was made public by https://reviews.llvm.org/D74158 due to Julia's use. @vtjnash

Reply:
Good point, I added an addDebugHandler method so that the Julia use case should continue to work.